app/src/main/res/layout/activity_login.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Maybe this view could be done simpler with a `ConstraintLayout`. What do you think?